data-collection-scripts/ltc-scraper/.gitignore

```diff
@@ -0,0 +1,106 @@
+# Logs
```

Comment:
this one's still here?

Reply:
Just a sec. Fighting with the GH Actions commit action, but I'll make sure it's gone